### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by ariana-bajrami** [vercel/community#7279] [Help] [answered]
### Summary

Hello Vercel Team

It has come to our attention that a domain hosted through your platform is conducting a phishing campaign against Pancakeswap

The malicious website is found at:  
hxxps://pancakesapv3[.]vercel[.]app/

The original website of Pancake Swap is found at:
https://pancakeswap.com/

Your assistance to suspend the website is highly appreciated.

### Example

https://pancakesapv3.vercel.app/

### Steps to Reproduce

...

**Comment by swarnava** [comment, marked as answer]
Hi,

Can you contact abuse@vercel.com? We can look into this.

**Comment by ariana-bajrami** [comment]
Hello Swarnava Sengupta,

Thank you for your response.

We already have reported to ***@***.***, however we have not heard
back from you.
Is there any way we can expedite the process?

Thank you,
Ariana Bajrami



On Tue, Jun 25, 2024 at 6:56 PM Swarnava Sengupta ***@***.***>
wrote:

> Hi,
>
> Can you contact ***@***.***? We can look into this.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/vercel/discussions/7279#discussioncomment-9873965>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AX7MTBSH7UKM7WVFZ555DULZJGOMRAVCNFSM6AAAAABJ34KKWCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TQNZTHE3DK>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 


*Ariana Bajrami*

Phishing Analyst

***@***.***

phishfort.com <https://www.phishfort.com/>

<https://www.linkedin.com/company/phishfort/>   <https://t.me/phishfort>
<https://twitter.com/PhishFort>

[a comment by an automated account is not shown]
